src/sentry/api/endpoints/group_details.py

```diff
@@ -203,6 +204,11 @@ def get(self, request: Request, group) -> Response:
                 inbox_reason = inbox_map.get(group.id)
                 data.update({"inbox": inbox_reason})
 
+            if "owners" in expand:
+                owner_details = get_owner_details([group], request.user)
+                owners = owner_details.get(group.id)
+                data.update({"owners": owners})
```

Comment:
If this is a regular thing we should consider including it in the serializer to avoid code duplication